bedrock/firefox/templates/firefox/whatsnew/fx64/whatsnew-fx64-base.html

```diff
@@ -32,6 +32,12 @@
   {% set sync_desc = _('Send an open tab on one device to all your others with a single tap. Much easier than texting or emailing yourself those links.') %}
 {% endif %}
 
+{% set fxa_utm_params = {
+  'campaign': 'wnp64',
+  'content': '/firefox/WNP/64/',
```

Comment:
Should this value be url encoded like the original?

Reply:
It is handled in the macro:

`{% do fxa_queries.append('utm_{}={}'.format(param, val.replace('/', '%2F'))) %}`